test/e2e/magellan-calypso.json

```diff
@@ -1,9 +1,8 @@
 {
-	"mocha_tests": [ "specs" ],
-	"mocha_args": "-R spec-junit-reporter",
+	"mocha_tests": [ "specs/specs-calypso" ],
 	"max_test_attempts": 3,
-	"max_workers": 6,
-	"suiteTag": "parallel;signup",
+	"max_workers": 1,
+	"suiteTag": "parallel",
```

Comment:
Everything tagged with `@signup` is also tagged with `@parallel`, therefore `@signup` is redundant.